### PR TITLE
Pretty comment

### LINKED ISSUE #104: Style the Comments

### PR DESCRIPTION
**Pull request by frinzekt**
closes #104 

Editing and Delete Options for the author of the comment
image

### CODE REVIEW

File: frontend/components/Comment.js [hunk 1]
```diff
@@ -8,40 +8,76 @@ class Comment extends Component {
...
   }
   componentDidMount() {
+    let elems = document.querySelectorAll('.fixed-action-btn')
+    let instances = M.FloatingActionButton.init(elems, { direction: 'left' })
     this.setState((prevState) => ({
       editValue: this.props.content,
     }))
   }
   render() {
+    console.log(this.props)
```
Comment (noobling) [inline frontend/components/Comment.js:18]: Please remoe this

File: frontend/components/Comment.js [hunk 2]
```diff
@@ -8,40 +8,76 @@ class Comment extends Component {
...
+                        }}
+                      >
+                        <i className="material-icons">mode_edit</i>
+                      </a>
+                    </li>
+                    {/* <li>
+                    <a className="btn-floating yellow darken-1">
+                      <i className="material-icons">info</i>
+                    </a>
+                  </li> */}
```
Comment (noobling) [inline frontend/components/Comment.js:62]: please remove comments

File: frontend/components/Comment.js [hunk 3]
```diff
@@ -8,40 +8,76 @@ class Comment extends Component {
...
+            </div>
+            <div
+              className="fixed-action-btn horizontal"
+              style={{ position: 'static' }}
+            >
+              {this.props.canEditOrDelete && (
+                <React.Fragment>
+                  <a className="btn-floating waves-effect waves-light ">
+                    <i className="large material-icons">more_vert</i>
+                  </a>
```
Comment (noobling) [inline frontend/components/Comment.js:44]: Is this used? Maybe remove it if it has no purpose

File: frontend/pages/post/[id].js [hunk 4]
```diff
@@ -255,10 +255,8 @@ class Comments extends React.Component {
             />
           </div>
         ) : (
-          <div>
-            <b>Please sign in</b>
-            <br />
-            <br />
+          <div className="container" style={{ marginBottom: 10 }}>
+            <b>Please sign in to add comment</b>
```
Comment (noobling) [inline frontend/pages/post/[id].js:259]: Would be nice to add a sign in link

File: frontend/components/Comment.js (hunk 3 above)
Comment (frinzekt) [reply frontend/components/Comment.js:44]: this one is used, this is the blue icon with :

**Pull request merged**